### ISSUE TITLE
🐛 [Bug] Segfault when compiling QuantConv2D+BatchNorm2D

### ISSUE THREAD

**Issue by narendasan** [pytorch/TensorRT#1575]
##  Bug Description

Segfault when running the attached repro script

## To Reproduce

```py

import torch
import torch.nn as nn
import trtorch
import pytorch_quantization
import os
print("PyTorch verison: ", torch.__version__)
print("TRTorch version: ", trtorch.__version__)
print("pytorch_quantization version", pytorch_quantization.__version__)
for envvar in ["CUDNN_VERSION", "TRT_VERSION", "PYTORCH_VERSION", "NVIDIA_PYTORCH_VERSION"]:
    print(f"{envvar}: {os.environ[envvar]}")

from pytorch_quantization import nn as quant_nn
quant_nn.TensorQuantizer.use_fb_fake_quant = True

size=[1,3,224,224]
net = nn.Sequential(
    quant_nn.QuantConv2d(3, 32, kernel_size=(1, 1), stride=(1, 1), padding=(1, 1), bias=False),
    nn.BatchNorm2d(32, eps=0.001, momentum=0.01, affine=True, track_running_stats=True),
).eval()
print(net)

ts_net = torch.jit.trace(net, torch.zeros(*size))
print("TorchScript success")
trt_net = trtorch.compile(
    ts_net, 
    {
        "inputs": [trtorch.Input(
            min_shape=[*size],
            max_shape=[*size],
            opt_shape=[*size],
            dtype=torch.float,
        )],
        "enabled_precisions": {torch.int8}, 
    })
trt_net(torch.zeros(*size).cuda())

```

## Expected behavior

This should not segfault

## Environment

> Build information about Torch-TensorRT can be found by turning on debug messages

PyTorch verison:  1.10.0a0+ecc3718
TRTorch version:  0.5.0a0
pytorch_quantization version 2.1.0
CUDNN_VERSION: 8.2.2.26
TRT_VERSION: 8.0.1.6+cuda11.3.1.005
PYTORCH_VERSION: 1.10.0a0+ecc3718
NVIDIA_PYTORCH_VERSION: 21.07

**Comment by narendasan** [comment]
@peri044 Can you take a look quickly and see if this is still an issue?

**Comment by peri044** [comment]
Tried with the latest code base and it runs successfully.
